x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/snapshotlifecycle/SnapshotLifecycleTaskTests.java

```diff
@@ -76,19 +81,22 @@ public void testSkipCreatingSnapshotWhenJobDoesNotMatch() {
                 .build())
             .build();
 
+        final SnapshotHistoryStore historyStore = mock(SnapshotHistoryStore.class);
```

Comment:
I think rather than mock the snapshot history store, we could subclass it and verify it with a real method without actually performing the indexing operation (I think mocks tend to make our tests more fragile)

Reply:
Done.